packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -35,7 +35,7 @@ const deserializeCookie = (input: Request | Response): string[] => {
 
 const serializeCookie = (input: string[]) => input.join(', ')
 
-export class Cookies extends Map<string, any> {
+export class Cookies extends Map<string, string> {
```

Comment:
Looks like we need to update the types further down for `store.get()`, we should probably also update `set(key: string, value: unknown` to `set(key: string, value: string`

Reply:
Seems like `set()` serializes to a string so that can remain as is 

https://github.com/vercel/next.js/blob/e4e7b07aa3bb6b39e94dd14e5c2daa2741594469/packages/next/server/web/spec-extension/cookies.ts#L18-L19